### PR TITLE
add mcps sample for galley tools

### PR DESCRIPTION
**Pull request by irisdingbj**
convenient for mcp server and client

### CODE REVIEW

File: galley/tools/mcps/main.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+)
+
+var (
+	serverAddr     = flag.String("server", "127.0.0.1:9901", "The server address")
+	kubeConfig     = flag.String("kubeConfig", "", "The path to kube configuration file")
+	configPath     = flag.String("configPath", "../../../tests/testdata/config", "The config files which galley will monitor")
+	meshConfigFile = flag.String("meshConfigFile", "../../../istioctl/cmd/istioctl/testdata/mesh-config.yaml", "The path for mesh config")
+)
+
+func main() {
```
Comment (ozevren) [inline galley/tools/mcps/main.go]: Why can't we use Galley itself for this purpose?

Comment (irisdingbj) [reply galley/tools/mcps/main.go]: To use galley locally, we need modify multiple parameters including probe and validation, etc. This is a minimum sample for running mcp galley server.

Comment (ozevren) [reply galley/tools/mcps/main.go]: Sounds like a batch file. :)

Comment (irisdingbj) [reply galley/tools/mcps/main.go]: OK, let me close this PR.

**Pull request closed without merging**